zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -1087,13 +1086,16 @@ where
                         Recipient::InternalAccount {
                             receiving_account: account,
                             external_address: None,
-                            note: PoolType::Shielded(ShieldedProtocol::Orchard),
+                            note: output_pool,
                         },
                         change_value.value(),
                         Some(memo),
                     ))
                 }
             }
+            PoolType::Transparent => {
+                return Err(Error::UnsupportedChangeType(output_pool));
```

Comment:
No, using the transparent `PoolType` for a `ChangeValue` is intentionally unsupported when "transparent-inputs" is disabled. The match arm was not previously needed because `output_pool` was of type `ShieldedProtocol`, and so this case was not expressible.

If "transparent-inputs" *is* supported, this match correctly does nothing, because that case is handled by the block at lines 1134 to 1173.